### PR TITLE
Fixes: #201 DropdownMenu list has no width when expanded thus it's not possible to see the options on IE11

### LINKED ISSUE #201: DropdownMenu list has no width when expanded thus it's not possible to see the options on IE11
### Prerequisites

- [X] Can you reproduce the problem in a [MWE](https://en.wikipedia.org/wiki/Minimal_working_example)?
- [X] Are you running the latest version?
- [X] Did you check the FAQs to see if that helps you?
- [X] Are you reporting to the correct repository?
- [X] Did you perform a search in the issues?

For more information, see the `CONTRIBUTING` guide.

### Versions

0.8.2-pre.593 and up

### Description

When using DropdownMenu on IE11, after clicking on the menu to see the list of options the list appears with 0 width, which makes it impossible to visualize or select any option

### Steps to Reproduce

1. Render a DropdownMenu component
2. Open the page with IE11
3. Expand the menu
4. See the result

**Expected behavior:** [What you expected to happen]

To see the list and be able to select any of the options

**Actual behavior:** [What actually happened]

MicrosoftTeams-image

**Environment details:** [OS, Browser, ...]

IE11

### Possible Solution
The issue was introduced in commit e47590ae14162eecaf0361f6d7ea1215d7843470 particularly in `ListItemContent` change from

```
const ListItemContent = styled.div`
  flex-grow: 1;
`;
```

to

```
const ListItemContent = styled.div`
 flex-grow: 1;
 flex: 1 0 0px;
`;r
```

The change adding flex-basis to 0px broke the IE11 compatibility.

### PR DESCRIPTION
**Pull request by gerkirill**
### Prerequisites

Please make sure you can check the following two boxes:

- [x] I have read the **CONTRIBUTING** document
- [x] My code follows the code style of this project

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [x] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
- [ ] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed

### Description

Fixes #201 - DropdownMenu and SelectButton when expanded - used to have an issue in IE11: the options list has had zero width.

### CODE REVIEW

**Comment by cz-white** [comment]
@dimabory  I am going to disagree with your comment here, the dropdown menu is a separate issue that (if it hasn't already) should be raised. 

#169 refers to the dropdownSelect component, not the dropdownMenu component which with the fix from @gerkirill, still works as expected.  

the issue you attached a screenshot of has to do with the dropdownMenu component specifically having no-wrap set to it, and with a max-width of 300px. I am assuming this is intentional to limit the size of such a menu, and the items in it. If we want the text to wrap here that component will need a larger refactor, as its width appears to be programmatically set, which leads to undesirable wrapping behavior. 

TLDR: I think @gerkirill s change fixes the issue at hand, does not impact #169 and a new issue may need to be raised for the dropdownMenu if it is decided to actually be an issue.

**Comment by guiherzog** [comment]
I also support @cz-white's opinion. Therefore, we will be merging this fix. Feel free to create an issue for the other problem.
